internal/pkg/cli/pipeline_init.go

```diff
@@ -647,8 +648,8 @@ func (o *initPipelineOpts) createPipelineManifest() error {
 		manifestMsgFmt = "Pipeline manifest file for %s already exists at %s, skipping writing it.\n"
 	}
 	log.Successf(manifestMsgFmt, color.HighlightUserInput(o.repoName), color.HighlightResource(o.manifestPath))
-	log.Infof(`The manifest contains configurations for your CodePipeline resources, such as your pipeline stages and build steps.
-Update the file to add additional stages, change the branch to be tracked, or add test commands or manual approval actions.
+	log.Debug(`The manifest contains configurations for your CodePipeline.
```

Comment:
Maybe keep the message generic here?
```suggestion
	log.Debug(`The manifest contains configurations for your pipeline.
```